docs/developer/attributes/configuration.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Attributes configuration
+sidebar_label: Configuration
+description: Learn how to configure attributes in Saleor.
+---
+
+### Attributes configuration
```

Comment:
**suggestion:**
We start the article with h3 headings and no prior introduction. Maybe we can add some sort of overview and then arrange two sections under "Configuration" heading?

## Overview (or whatever)

...

## Configuration

### Attributes 

...

### Product types

or at least change the headings to h2 (so it's consistent with the rest)